README.md

```diff
@@ -354,30 +354,30 @@ The fourth cache behaviour handles next API requests `api/*`.
-| verbose                  | `boolean`         | `false`                                                            | Print verbose output to the console.                                                                                                                                                                                                                                 |
+| Name                     | Type              | Default Value                                                      | Description                                                                                                                                                                                                                                                                                                                                     |
+| ------------------------ | ----------------- | ------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| domain                   | `Array`           | `null`                                                             | For example `['admin', 'portal.com']`                                                                                                                                                                                                                                                                                                           |
+| bucketName               | `string`          | `null`                                                             | Custom bucket name where static assets are stored. By default is autogenerated.                                                                                                                                                                                                                                                                 |
+| bucketRegion             | `string`          | `null`                                                             | Region where you want to host your s3 bucket. Make sure this is geographically closer to the majority of your end users to reduce latency when CloudFront proxies a request to S3. Set this if you are experiencing 307 temporary redirects. See https://aws.amazon.com/premiumsupport/knowledge-center/s3-http-307-response/ for more details. |
```

Comment:
The addition of "Set this if you are experiencing ... etc." should be deleted. That has been solved now.

Reply:
Fixed